gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/writer/HiveMetadataWriter.java

```diff
@@ -292,9 +292,9 @@ private void schemaUpdateHelper(GobblinMetadataChangeEvent gmce, HiveSpec spec,
       return;
     }
     //Force to set the schema even there is no schema literal defined in the spec
-    spec.getTable()
-        .getSerDeProps()
-        .setProp(AvroSerdeUtils.AvroTableProperties.SCHEMA_LITERAL.getPropName(), latestSchemaMap.get(tableKey));
+    if(latestSchemaMap.containsKey(tableKey)) {
```

Comment:
This change looks unrelated to the PR.

Reply:
Thanks for catching that. just revert the change, will create a separate pr for that fix.